checklist.html.md.erb

```diff
@@ -231,6 +232,10 @@ If you are not using PCF Healthwatch, you can do some or all of the following to
 
 Check that a test app can be pushed and scaled horizontally, manually or through automated testing. This check ensures that the platform supports apps as expected before the upgrade.
 
+### <a id="validate-installed-buildpacks"></a> Validate Installed Buildpacks
+
+Buildpacks may now have an [associated stack](https://docs.cloudfoundry.org/buildpacks/stack-association.html) denoting which stacks they are compatible with. During this transition, PAS will begin installing multiple versions of buildpacks for different stacks (e.g. `cflinuxfs2`, `cflinuxfs3`, etc.). To ensure that the buildpacks are [installed without errors](https://community.pivotal.io/s/article/pcf-upgrade-fails-with-stacklessandstackfulmatchingbuildpacksexisterror), if you have multiple buildpacks installed with the same name all buildpacks with that name must be associated with a stack. Additionally, any buildpacks that do not have an associated stack should be unlocked prior to upgrading. Refer to the [following documentation](https://docs.cloudfoundry.org/buildpacks/stack-association.html#stack-cli) for more information on associating a stack with an existing buildpack.
```

Comment:
@tylerphelan @sclevine

From the buildpacks side of things, what do you think of this explanation?

Reply:
Another helpful  link: https://community.pivotal.io/s/article/errands-in-upgrade-to-pcf-or-pas-2-3-are-failing-as-deploying-on-a-stack-is-not-supported-by-the-buildpack

Reply:
@tylerphelan is that necessary in 2.2? Or just 2.3+?

Thanks to you both, I will add that link to the list of KB articles above and incorporate the custom/non-default buildpacks suggestion.

Reply:
@ljarzynski 

The suggestions above don't apply to this version of the upgrade checklist (for PAS 2.2), but I will be including them in followup PRs for PAS 2.3 and 2.4.

I also flew this change past @rhall-pivotal and he thinks it looks good, so I don't expect to make additional updates in this PR.